xml/System.Drawing/Pen.xml

```diff
@@ -260,13 +260,13 @@
       </Parameters>
       <Docs>
         <param name="color">A <see cref="T:System.Drawing.Color" /> structure that indicates the color of this <see cref="T:System.Drawing.Pen" />.</param>
-        <param name="width">A value indicating the width of this <see cref="T:System.Drawing.Pen" />.</param>
+        <param name="width">A value indicating the width of this <see cref="T:System.Drawing.Pen" />.  A value of 0 will be treated similarly to a value of 1, except that the `width` will not be affected by scale-transform operations that are in effect for the Graphics object that the <xref:System.Drawing.Pen> is used for, resulting in the width to be always 1 pixel.</param>
```

Comment:
This is a bit long for a parameter description and I think belongs in the remarks instead.

Reply:
Thanks for the feedback...Updated the remarks section.